dev/integration_tests/flutter_gallery/lib/demo/material/bottom_app_bar_demo.dart

```diff
@@ -323,9 +323,7 @@ class _Heading extends StatelessWidget {
       alignment: AlignmentDirectional.centerStart,
       child: Text(
         text,
-        style: theme.textTheme.bodyText2.copyWith(
-          color: theme.primaryColor,
-        ),
+        style: theme.textTheme.bodyText1,
```

Comment:
I assume that a change like this works without the color change for both dark and light modes?

Reply:
It does! All three themes (gallery, light, and dark) passed.